### DISCUSSION TITLE
How to remove these red lines?

### DISCUSSION THREAD

**Discussion by bhuppidhamii** [redhat-developer/vscode-java#2817] [Q&A]
![image](https://user-images.githubusercontent.com/54303662/204296792-a5c2be0d-6df4-416e-b55d-939d76eae768.png)

**Comment by fbricon** [comment]
see the Problems view in VS Code.
 
* The revArrayfun method expects you return an int, but nothing is returned. Either return an int or change the return type to void.
* numbers[]= (....) is not valid Java syntax. You probably want to declare
> int[] numbers = {2, 4, 6, 8, 10};
